### DISCUSSION TITLE
disabling filtering for 2d pixel games

### DISCUSSION THREAD

**Discussion by petran79** [flyinghead/flycast#1292] [Q&A]
I recall that when I connected the Dreamcast to a LCD TV via VGA cable, 2d pixel games like Gunbird 2 or Street Fighter Grand Master Challenge were very blocky and pixellated.

However when I choose vga output on Flycast, Gunbird 2 has automatic smooth filter, which is not how it looks on original Dreamcast at all via VGA. 

Is there a way to disable filtering completely for similar games relying on pixels?

I'd like to experience with a vga monitor at 240p that has its own native scanlines.

**Comment by flyinghead** [comment]
The rendered image is displayed using linear filtering to the screen or window. You can force filtering to Nearest in `Settings > Video > Texture Filtering > Force Nearest-Neighbor`.
However this will also affect *all* textures in the game, which may have unexpected consequences.
Another way to avoid linear filtering is to have a window or screen with the same resolution as expected by the original Dreamcast.
